### ISSUE TITLE
Issue with Pre configured image

### ISSUE THREAD

**Issue by lucashou** [brutella/hkcam#107]
HI 

Just flashed my SD card with the pre configured imaged.
Not sure it should have been there, but the /Volumes was not there, so I created it and added the SSID elements.
I Inserted the SD card inside the PI 0 W and booted it 

So far nothing on the home kit, and on the PI screen it's kinda stuck at the boot Login/PWD, is that normal ? 

thanks for your help 

(Sorry if it's idiot i'm kind new to this world)

**Comment by MunichTester** [comment]
No worries. I guess you need to login with the standard pi/raspberry account first. Then you need to setup wifi manuell too.

**Comment by lucashou** [comment]
Alright thanks 

Do you have the instructions to set up manullay the wifi ? 

Lucas

**Comment by MunichTester** [comment]
Here you can find these infos https://www.raspberrypi.org/documentation/configuration/wireless/wireless-cli.md

**Comment by lucashou** [comment]
hi , 

thanks a lot . I now have my pi on the Network , but the home app cannot detect anything ... :(

**Comment by nickjvturner** [comment]
If your pi is inside the 3D printed case? Watch out for the camera sensor ribbon getting displaced by the lid.

Either way, your next step is to remote into the pi with ssh and view the logs.

**Comment by lucashou** [comment]
Hi, 

The board is still outside of the enclosure. I checked the camera connexion ,seems just fine. Any idea which file to look at in particular ?

**Comment by nickjvturner** [comment]
Give this a try, ssh into the pi.

Type out:
`tail -f /var/log/hkcam/current`

This should display the hkcam logs, as new logs are made, they will be displayed.

To exit this view hit:
`ctrl + c`

If you copy paste the output from the hkcam logs into here, someone more knowledgeable than me might be able to point you in the right direction.

**Comment by lucashou** [comment]
Hi,

The log says Error  : Failed to Open /dev/videoO : so such file or directory

Thanks for the help

**Comment by cswelin** [comment]
> The log says Error : Failed to Open /dev/videoO : so such file or directory

I"m see the same error as well.

my dev/video looks as follows.
```
ls -l /dev/video*
crw-rw---- 1 root video 81, 0 Feb  5 21:16 /dev/video1
crw-rw---- 1 root video 81, 1 Feb  5 21:16 /dev/video10
crw-rw---- 1 root video 81, 2 Feb  5 21:16 /dev/video11
crw-rw---- 1 root video 81, 3 Feb  5 21:16 /dev/video12
````

**Comment by RomeHein** [comment]
Yep same here with the latest image

**Comment by brutella** [comment]
Please check your ribbon cables when connecting a standard camera module.

FYI: I've just released version [0.1.0](https://github.com/brutella/hkcam/releases/tag/v0.1.0) of hkcam.
